### PR TITLE
Add command: randomSampleFromInput

### PR DESCRIPTION
**Pull request by tannerwelsh**
Adds a new command `randomSampleFromInput` which takes a comma-separated set of values as input and returns a random sample from that list.

Basically the same as [_.sample](https://lodash.com/docs/4.17.11#sample).

_Example input_: `apples,12893,_Captain Marvel_`

`randomSampleFromInput` would randomly return one of:

- `apples`
- `12893`
- `_Captain Marvel_`

randomsample

### CODE REVIEW

**Comment by tannerwelsh** [comment]
Hey @jrebocho! Love your package. I found myself today wanting to get a random sample from a pre-defined list, and so I built this command. What do you think? Does it fit within the scope of your package?

File: tests/validations.test.js [hunk 1]
```diff
@@ -116,4 +121,32 @@ describe('>>>>> Validator Functions Tests', () => {
       expect(isValidIntRange(valueToTest)).toBeFalsy()
     })
   })
+
+  describe('.isStringWithComma', () => {
+    it('validates strings containing at least one comma', () => {
```
Comment (jrebocho) [inline tests/validations.test.js]: Can we split each one of this tests into different `it` clauses?  I prefer to have only one `expect` per test (although this tests are really simple and small 😅)

**Comment by jrebocho** [comment]
Hi @tannerwelsh ! Thank you so much for your contribution! This new feature absolutely fits on the scope of this package and we can definitely put it into the next release!

File: tests/validations.test.js (hunk 1 above)
Comment (tannerwelsh) [reply tests/validations.test.js]: Absolutely! Will post a new commit in a sec...

**Comment by tannerwelsh** [comment]
Thanks for the helpful feedback @jrebocho! Made an update to the test file in 9ecce67, let me know if that works for you.

**Comment by tannerwelsh** [comment]
Thanks @jrebocho! Looking forward to it :)

**Pull request merged**